include/ap_common.h

```diff
@@ -59,7 +59,7 @@ class half;
 // for fprintf messages.
 #include <stdio.h>
```

Comment:
Probably irrelevant, but here assert.h and stdio.h could be maybe changed to cassert and cstdio
include/etc/ap_private.h has also an occurrence of stdio.h